private/rules/maven_install.bzl

```diff
@@ -27,7 +27,8 @@ def maven_install(
         aar_import_bzl_label = DEFAULT_AAR_IMPORT_LABEL,
         duplicate_version_warning = "warn",
         repin_instructions = None,
-        ignore_empty_files = False):
+        ignore_empty_files = False,
+        resolver_additional_options = []):
```

Comment:
`coursier_additional_options`, please.

Reply:
Sure. I was not sure whether it was fine to expose `coursier` in the parameter name here. Will fix.